api/swagger.yaml

```diff
@@ -210,6 +210,44 @@ definitions:
       PathInContainer: "/dev/deviceName"
       CgroupPermissions: "mrw"
 
+  DeviceRequest:
```

Comment:
Could you add a mention to the API changelog; https://github.com/moby/moby/blob/master/docs/api/version-history.md#v140-api-changes

This affects endpoints `POST /containers/create`, and `GET /containers/{id}/json`